### PR TITLE
Fix: API Endpoint

### PR DESCRIPTION
**Pull request by miklb**
## Description
This pull request replaces the API endpoint with the production database. It also exposes the map zoom to config file.

Addresses issue #76

## List of changes

- [  Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

Breaking in that existing deploys will default to the zoom set in the theme config. Dorian needs to be re-centered on the map and can update zoom at same time.

## Checklist:
- [x] This pull request relates to a ticket.
- [x] This code is tested.
- [x] I want my code added to the response theme.

### CODE REVIEW

**Comment by miklb** [comment]
@plocket could you take a look at this PR and let us know what you think? I know you can't approve PRs but your contributions and opinion matters.

**Comment by plocket** [comment]
Turns out I can approve PRs! If you prefer I can remove the approval, though, and let one of the long-standing contributors have at it.

**Comment by plocket** [comment]
One caveat - this does handle the zooming, but there still isn't really different types of data to test with. Maybe the title of that issue could be changed to more specifically say that it's fixing the fetch error/page not loading when trying to access the Florence endpoint.

**Comment by miklb** [comment]
No, it doesn’t fix types of data, but it does fix endpoint & provide live data to continue to allow as much participation as possible for now.

File: _config.yml [hunk 1]
```diff
@@ -34,14 +34,15 @@ plugins:
...
-# TODO: use config file for map center
+
 mapbox_access_token: pk.eyJ1IjoibWlrbGItYzR0YiIsImEiOiJjam5kOGg3c2kwMDB3M3dwYXQyMDJpdzFtIn0.yKOyo193qF1quWsNwiFzug
 
-shelter_map_center: -88.9159, 35.8198
+shelter_map_center: -95.7129, 37.0902
+shelter_zoom: 3.6
 
-pod_map_center: -88.9159, 35.8198
+pod_map_center: -95.7129, 7.0902
```
Comment (nihonjinrxs) [inline _config.yml]: This center point doesn't match shelters  -- is that right?

Comment (nihonjinrxs) [reply _config.yml]: Likely should be `37.0902` instead of `7.0902`

Comment (miklb) [reply _config.yml]: The default center I used was middle of US and a lower zoom so when there isn't an event we can pick up any open shelters for development. For an event, we can adjust the centers and zoom in the config.

Comment (miklb) [reply _config.yml]: yep, sorry.

**Pull request merged**